modules/aws/lambda.go

```diff
@@ -2,51 +2,136 @@ package aws
 
 import (
 	"encoding/json"
+	"errors"
 	"fmt"
 
 	"github.com/aws/aws-sdk-go/service/lambda"
 	"github.com/gruntwork-io/terratest/modules/testing"
 	"github.com/stretchr/testify/require"
 )
 
+// LambdaOptions contains additional parameters for InvokeFunctionWithParams().
+// It contains a subset of the fields found in the lambda.InvokeInput struct.
+type LambdaOptions struct {
+	// FunctionName is a required field containing the lambda function name.
+	FunctionName *string
```

Comment:
NIT: Since this is actually required, I'd rather keep it as a separate parameter in the function signature, rather than an optional value in the "options" `struct`.

Reply:
Done.